tests/channel_int/channel_int.c

```diff
@@ -21,6 +21,9 @@
 #include "c_util/rc_ptr.h"
 #include "c_util/channel.h"
 
+
+#define DISABLE_TEST_FUNCTION(x) static void x(void)
```

Comment:
We have this in c test runner switcher now, so no need for it anymore.

Reply:
Removed.